mock matchers › failing output › expect.mock.not.toBeCalled overflow print
<dim>expect.mock(</dim><red>mock</red><dim>).</dim>not<dim>.toBeCalled(</dim><green></green><dim>)</dim>

Expected the mock not be called, but it was called with:
  <red>{10, 11}</red>, <red>{7, 8}</red>, <red>{5, 6}</red><red> and two more calls.</red>
